src/symfony/harness/attributes/docker.yml

```diff
@@ -0,0 +1,9 @@
+attributes:
+  services:
+    php-base:
+      environment:
+        APP_DEBUG: "0"
+        APP_ENV: "= (@('app.mode') == 'development' ? 'dev' : 'prod')"
+      environment_secrets:
+        APP_SECRET: = @('symfony.secret')
```

Comment:
I think it might be a bit confusing having a `symfony.secret` that is only for local environment

Reply:
Removed